### DISCUSSION TITLE
怎样评定训练虚度

### DISCUSSION THREAD

**Discussion by TaChao** [zai-org/ChatGLM3#924] [General]
单机单卡的时候日志输出
1000/1000 [10:56<00:00, 13.34it/s]
单机八卡的时候输出 
1000/1000 [05:49<00:00, 11.60it/s]

看it/s 的数据反而性能下降了一点。这个its是输出的单卡的速度吗？

看推理时间 只提高了50% 也没有达到 8卡应该有的速度

**Comment by zRzRzRzRzRzRzR** [comment]
那个值确实是速度的意思

8卡用deepspeed加速吧

**Reply by TaChao** [reply]
是用deepspeed。 这个速度在8卡下是指总速度还是单卡速度

**Reply by zRzRzRzRzRzRzR** [reply]
？那这不对吧，应该是每张卡都是这个单卡的速度

**Reply by TaChao** [reply]
如果每张卡都是11.60it/s 但是看他的计算时间 只减少了一半，也很奇怪
